src/ArduinoIoTCloud.h

```diff
@@ -43,14 +43,12 @@ typedef struct {
 extern ConnectionManager *ArduinoIoTPreferredConnection;
 
 enum ArduinoIoTConnectionStatus {
```

Comment:
@lxrobotics 
I've been looking up `enum class` and I think it's a great way to go, but not for this PR.
We're awaiting on this merge to move forward and release MKR GSM support which after this is merged in will be ready.
this will definitely end up in the next update :)